shoop_tests/admin/test_product_module.py

```diff
@@ -34,12 +35,15 @@ def test_product_module_search(rf, admin_user):
 
 @pytest.mark.django_db
 def test_product_edit_view_works_at_all(rf, admin_user):
-    get_default_shop()
+    shop = get_default_shop()
+    default_product = get_default_product()
```

Comment:
As mentioned in :meat_on_bone: :space_invader: :

If you're going to mutate the visibility limit, it'd be better to create a new product for testing